docs/UserGuide.md

```diff
@@ -330,12 +331,20 @@ Adds a single tuition session to TutorBuddy.
 Format: `add_session n/STUDENT_NAME d/DATE t/TIME k/DURATION s/SUBJECT f/FEE`
 
 * `STUDENT_NAME` should match the exact student’s name in TutorBuddy
+* `DATE` should be the date of the session in `yyyy-mm-dd` format
 * `DURATION` should be in minutes
-* `FEE` should be the total tuition fee for the total duration
+* `FEE` should be the total tuition fee for 1 session of the given duration
+    * TutorBuddy only accepts `FEE` within an acceptable range of values between 0 (Voluntary Session) to below 10000 as we do not expect a single session to cost $10,000 or more.
```

Comment:
Not sure if we should rename voluntary session to **free** instead? While reading it, thought it was a new kind of session HAHA

Also, not sure if we should justify why we chose 0 - $10,000, feels like we're doing it just not to get penalised HAHA maybe specifying the range is enough. either way works!

Reply:
Actually on second thought after reading your codes below: maybe can rephrase it to, accepts `FEE` within an acceptable range of values between 0 to 9999.99 (both inclusive)

Reply:
Updated to "TutorBuddy only accepts `FEE` within an acceptable range of values between 0 and 999999.99 (both inclusive) as we do not expect a single session to cost more than $999,999.99 or more." as per suggestion.